docs/source/markdown/podman-container-restore.1.md

```diff
@@ -95,6 +95,14 @@ This option must be used in combination with the **--import, -i** option.
 When restoring containers from a checkpoint tar.gz file with this option,
 the content of associated volumes will not be restored.
 
+#### **--publish**, **-p**
+
+Replaces the ports that the container publishes, as configured during initial
+container start, with a new set of port forwarding rules.
+
+For more details please see **podman run --publish**.
+
+
```

Comment:
it would be good to add an example, and we probably should add `podman-run(1)` to the See Also list below